### PR TITLE
Test Python 3.10 on daint

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

**Comment by havogt** [comment]
bors try

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -53,6 +59,12 @@ build py39:
   variables:
     <<: *py39
 
+build py310:
+  allow_failure: true
+  extends: build py38
+  variables:
+    <<: *py39
```
Comment (havogt) [inline .gitlab-ci.yml:66]:
```suggestion
    <<: *py310
```

**Comment by havogt** [comment]
bors try

**Comment by havogt** [comment]
bors try-

**Comment by havogt** [comment]
bors try

**Review by gronerl — CHANGES_REQUESTED** [review]

**Comment by havogt** [comment]
bors try

**Review by gronerl — APPROVED** [review]

**Pull request merged**